FCWTAPI/PlottingUtils.cs

```diff
@@ -41,15 +41,15 @@ public class PlottingUtils
         /// Functionality will be added to scale axes with time and frequency info from CWT
         /// </summary>
         /// <param name="data"> Input double[,] to plot</param>
-        /// <param name="title">Plot title</param>
+        /// <param name="plotTitle">String containing the desired plot title</param>
         /// <returns></returns>
-        public static PlotModel GenerateHeatMap(double[,] data, string title)
+        public static PlotModel GenerateHeatMap(double[,] data, string plotTitle)
```

Comment:
The plot types are limited to 3 for the XY plots, but I wanted to allow any string to be passed through as a title.

Reply:
I don't really like using strings in method parameters. Include an "Any" option with the enum and add a nullable string (string?) method parameter. Then it can be an enum with the option to pass any string as a title.

Reply:
Okay, sounds good.